applications/DamApplication/dam_application_variables.cpp

```diff
@@ -47,4 +47,8 @@ namespace Kratos
     KRATOS_CREATE_VARIABLE( Matrix, INITIAL_NODAL_CAUCHY_STRESS_TENSOR )
     KRATOS_CREATE_VARIABLE( double, PLACEMENT_TEMPERATURE )
 
+    // From Solid
+    KRATOS_CREATE_3D_VARIABLE_WITH_COMPONENTS( FORCE_LOAD )
+    KRATOS_CREATE_VARIABLE(bool, COMPUTE_CONSISTENT_MASS_MATRIX)
```

Comment:
This variable is defined in the Structural?

Reply:
I don’t think so. I checked whether it was defined there and it is not (if I am not mistaken at least)